settings.py

```diff
@@ -0,0 +1,14 @@
+from pydantic.v1 import BaseSettings
+
+
+class Setting(BaseSettings):
+    PROJECT_NAME: str = "FastAPI City Temperature Management API"
+
+    DATABASE_URL: str | None = "sqlite+aiosqlite:///./city_temperature_management.db"
```

Comment:
The `DATABASE_URL` is set with a default value for a SQLite database. Consider using environment variables to override this in different environments (e.g., development, testing, production) to ensure flexibility and security.